introspection-engine/introspection-engine-tests/src/test_api.rs

```diff
@@ -248,7 +285,7 @@ impl TestApi {
     }
 
     fn pure_config(&self) -> String {
-        format!("{}\n{}", &self.datasource_block(), &self.generator_block())
+        format!("{}\n{}", &self.datasource_block_string(), &self.generator_block())
```

Comment:
I think `DatasourceBlock`'s display impl could have the namespaces, but it's complicated and I am probably missing some part of the picture. Why do we want the new method?